hibernate-core/src/main/java/org/hibernate/type/descriptor/java/ClobJavaType.java

```diff
@@ -128,6 +128,21 @@ else if ( String.class.isAssignableFrom( type ) ) {
 		throw unknownUnwrap( type );
 	}
 
+	private Clob getOrCreateClob(Clob value, WrapperOptions options) throws SQLException {
+		if(options.getDialect().useConnectionToCreateLob()) {
+			if(value.length() == 0) {
+				// empty Clob
+				return options.getLobCreator().createClob("");
+			}
+			else {
+				return options.getLobCreator().createClob(value.getSubString(1, (int) value.length()));
```

Comment:
```suggestion
		if ( options.getDialect().useConnectionToCreateLob() ) {
			return value.length() == 0
					// empty Clob
					? options.getLobCreator().createClob("");
					: options.getLobCreator().createClob( value.getSubString( 1, (int) value.length() ) );
```

Reply:
That one will not work because of the trailing semicolon:
`? options.getLobCreator().createClob("");`

Reply:
sorry, fixed now